cmd/put.go

```diff
@@ -103,7 +117,9 @@ func init() {
 	// Cobra supports local flags which will only run when this command
 	// is called directly, e.g.:
 	putCmd.Flags().StringP("kms-key-id", "k", "", "KMS CMK ID to use for encryption")
+	putCmd.Flags().BoolP("output-version-id", "o", false, "Set to output the version id of the uploaded object")
```

Comment:
I'd rather it just be something like "verbose output" like `-V`. That way, it can be extended to other outputs, if desired in the future

Reply:
Note, that's a capital V